src/monovec.jl

```diff
@@ -67,7 +67,8 @@ const DMonoVecElemNonConstant{C} = Union{PolyVar{C}, Monomial{C}, Term{C}}
 const DMonoVecElem{C} = Union{Int, DMonoVecElemNonConstant{C}}
 const DMonoVec{C} = AbstractVector{<:DMonoVecElem{C}}
 
-MP.emptymonovec(vars::VarVec{C}) where {C} = MonomialVector{C}(vars, Vector{Int}[])
+MP.emptymonovec(vars::NTuple{N, PolyVar{C}}) where {N, C} = MonomialVector{C}(collect(vars), Vector{Int}[])
```

Comment:
It doesn't work without the `collect` ? So in addition of not being tested, it didn't work before this PR ?

Reply:
Right, if I didn't misunderstand what the output is supposed to be... and I assumed `NTuple{<:Integer, PolyVar{C}}` was meant to be `NTuple{N, PolyVar{C}}`. Currently,
```julia
julia> emptymonovec((x,y))
0-element MonomialVector{true}
```
Since `emptymonovec` isn't implemented in TypedPolynomials and is only called in Dynamic polynomials in https://github.com/JuliaAlgebra/DynamicPolynomials.jl/blob/f8c3d074fdaab8a88412ae7154db50d875ff5073/src/poly.jl#L36 
that didn't cause any trouble.

Reply:
Then you can remove it, it indeed doesn't work on master
```
julia> @polyvar x y

(x, y)

julia> 

julia> emptymonovec((x, y))
ERROR: MethodError: no method matching monomialtype(::Tuple{PolyVar{true},PolyVar{true}})
Closest candidates are:
  monomialtype(::Type{#s26} where #s26<:Union{Monomial{C}, PolyVar{C}, Polynomial{C,T} where T, Term{C,T} where T}) where C at /home/blegat/.julia/dev/DynamicPolynomials/src/DynamicPolynomials.jl:18
  monomialtype(::Union{Monomial{C}, PolyVar{C}, Polynomial{C,T} where T, Term{C,T} where T}) where C at /home/blegat/.julia/dev/DynamicPolynomials/src/DynamicPolynomials.jl:19
  monomialtype(::Union{Type{M<:AbstractMonomial}, M<:AbstractMonomial}) where M<:AbstractMonomial at /home/blegat/.julia/packages/MultivariatePolynomials/gLVa3/src/monomial.jl:12
  ...
Stacktrace:
 [1] emptymonovec(::Tuple{PolyVar{true},PolyVar{true}}) at /home/blegat/.julia/packages/MultivariatePolynomials/gLVa3/src/monovec.jl:10
 [2] top-level scope at none:0
```

Reply:
Should it be removed from MultivariatePolynomials  as well? Since MultivariatePolynomials is reexported, `emptymonovec((x, y))` would throw an error from the fallback method
https://github.com/JuliaAlgebra/MultivariatePolynomials.jl/blob/master/src/monovec.jl#L10
(`monomialtype` wouldn't be defined for `(x,y)`).

Reply:
This function is not defined for tuples of variables in the doc of MultivariatePolynomials. It does work if you give it at polynomial, term, ... though so it should not be removed IMO